documentation.html

```diff
@@ -56,11 +56,109 @@
     <main class="container">
         <div class="docs-section">
             <div class="docs-container">
+                <div id="buttons">
+                    <h1 class="md-title">Buttons</h1>
+                    <p class="rg-p mg-t-10">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Aspernatur deserunt id ipsam doloremque
+                        sapiente voluptates facere necessitatibus ex debitis incidunt a, tempore, quas iste eveniet
+                        facilis minus. Perferendis, fugit. Aliquid?
 
-                <!-- for example -->
-                <!-- <div class="compcontainer"></div> -->
+                    </p>
 
+                    <div class="mg-t-30">
+                        <h2 class="rg-booky-title">Solid Buttons</h2>
+                        <div class="mg-t-30 btn-container">
+                            <div class="btn-examples">
+                                <button class="btn primary-btn solid">Primary Button</button>
+                                <button class="btn secondary-btn solid">Secondary Button</button>
+                                <button class="btn success-btn solid">Success Button</button>
+                                <button class="btn warning-btn solid">Warning Button</button>
+                                <button class="btn danger-btn solid">Error Button</button>
+                            </div>
+                        </div>
+                        <div class="code-example">
+                            <iframe class="code-snip"
+                                src="https://carbon.now.sh/embed?bg=rgba%28189%2C213%2C234%2C0%29&t=vscode&wt=sharp&l=auto&width=859&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=18px&lh=144%25&si=false&es=2x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2520primary-btn%2522%253EPrimary%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520secondary-btn%2522%253ESecondary%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520success-btn%2522%253ESuccess%2520Button%253C%252Fbutton%253E%2520%250A%253Cbutton%2520class%253D%2522btn%2520warning-btn%2522%253EWarning%2520Button%253C%252Fbutton%253E%250A%253Cbutton%2520class%253D%2522btn%2520danger-btn%2522%253EError%2520Button%253C%252Fbutton%253E"
+                                style="width: 677px; height: 204px; border:0; transform: scale(1); overflow:hidden;"
+                                sandbox="allow-scripts allow-same-origin">
```

Comment:
you can remove the inline styling here and shift to the css file. Also the iframe are same for the examples

Reply:
okay will change it